Jellyfin.Server/Migrations/MigrationRunner.cs

```diff
@@ -88,6 +88,12 @@ public static void RunPreStartup(ServerApplicationPaths appPaths, ILoggerFactory
                  ? (MigrationOptions)xmlSerializer.DeserializeFromFile(typeof(MigrationOptions), migrationConfigPath)!
                  : new MigrationOptions();
 
+            // 10.10 specific EFCore migration check.
+            if (migrationOptions.Applied.Any(f => f.Id.ToString("D") == "5C4B82A2-F053-4009-BD05-B6FCAD82F14C" || f.Name == "MigrateUserDatabase"))
```

Comment:
MigrateUserDatabase was applied in 2020. Won't you be bricking all existing 10.10 installs with this?